arm-software/ci/automerge.py

```diff
@@ -166,6 +166,12 @@ def pr_exist_for_label(project_name: str, label: str) -> bool:
     return len(json.loads(gh_process.stdout)) > 0
 
 
+def is_worktree_clean(git_repo: Git) -> bool:
+    # `git status --porcelain` returns an empty result if worktree is clean
+    status_output = git_repo.run_cmd(["status", "--porcelain"]).strip()
+    return len(status_output) > 0
```

Comment:
An empty result means the worktree is clean? But here you seem to be returning `True` if the result is _not_ empty.

Reply:
🤦 
Thanks for catching this! It should be fixed now.